### PR TITLE
Add automatic release each day

### PR DESCRIPTION
**Pull request by ILoveMyTortoise**

### CODE REVIEW

File: .github/workflows/automatic-release.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    branches:
+      - "main"
+
+jobs:
+  pre-release:
+    name: "Pre Release"
+    runs-on: "ubuntu-latest"
+
+    steps:
+      - name: Get next version
```
Comment (rochar) [inline .github/workflows/automatic-release.yml]: we are using gitversion for computing, can we use the same

File: .github/workflows/automatic-release.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      - name: "Build & test"
+        run: |
+          echo "done!"
+
+      - uses: "marvinpinto/action-automatic-releases@latest"
+        with:
+          repo_token: "${{ secrets.GITHUB_TOKEN }}"
+          automatic_release_tag: "${{ steps.versioning.outputs.version }}"
+          prerelease: false
+          title: "${{ steps.versioning.outputs.version }}"
```
Comment (rochar) [inline .github/workflows/automatic-release.yml]: add the v to version

**Review by rochar — CHANGES_REQUESTED** [review]

File: .github/workflows/automatic-release.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+name: "Scheduled release"
+
+permissions: write-all
+
+on:
+  schedule:
+    # Each day at 01:00 UTC
+    - cron:  '0 1 * * *'
+  workflow_dispatch:
+  push:
```
Comment (rochar) [inline .github/workflows/automatic-release.yml]: we can remove push to main for now

File: .github/workflows/automatic-release.yml (hunk 1 above)
Comment (ILoveMyTortoise) [reply .github/workflows/automatic-release.yml]: done

**Review by rochar — APPROVED** [review]

**Pull request merged**